### PR TITLE
Set cursor on hover.

### PR DESCRIPTION
**Pull request by sheeley**
Currently, mouse and keyboard interactions are separate. If you navigate using
the keyboard, then hover with the mouse, the cursor remains on the suggestion
the keyboard had navigated to.

With this change, the selected suggestion is set on hover, so hovering and then
pressing a key will use the hovered suggestion as its starting point.

### CODE REVIEW

**Comment by ApfelUser** [comment]
LGTM

/cc @corejavascript/collaborators

File: src/typeahead/menu.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ var Menu = (function() {
 
       onSelectableClick = _.bind(this._onSelectableClick, this);
       this.$node.on('click.tt', this.selectors.selectable, onSelectableClick);
+      this.$node.on('mouseover', this.selectors.selectable, function(){ that.setCursor($(this)) })
```
Comment (jcrben) [inline src/typeahead/menu.js]: Probably best to drop a semicolon here to keep the style consistent.

**Comment by jcrben** [comment]
:+1:

**Comment by sheeley** [comment]
@jcrben added semicolon, also ran `grunt` to update dist files.

**Comment by sheeley** [comment]
Is the build expected to complete? It appears to have failed installing bower requirements.

**Pull request merged**